commands/system.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+var SystemCmd = &cobra.Command{
+	Use:   "system",
+	Short: "System management",
+	Long:  `System management commands for interacting with the server state and configuration.`,
+}
+
+var SystemGetBusyCmd = &cobra.Command{
+	Use:     "getbusy",
+	Short:   "Get the current busy state",
+	Long:    `Gets the server busy state (high load) and timestamp corresponding to when the server busy flag will be automatically cleared.`,
+	Example: `  system getbusy`,
+	Args:    cobra.NoArgs,
+	RunE:    withClient(getBusyCmdF),
+}
+
+var SystemSetBusyCmd = &cobra.Command{
+	Use:     "setbusy -s [seconds]",
+	Short:   "Set the busy state to true",
+	Long:    `Set the busy state to true for the specified number of seconds, which disables non-critical services.`,
+	Example: `  system setbusy -s 3600`,
+	Args:    cobra.NoArgs,
+	RunE:    withClient(setBusyCmdF),
+}
+
+var SystemClearBusyCmd = &cobra.Command{
+	Use:     "clearbusy",
+	Short:   "Clears the busy state",
+	Long:    `Clear the busy state, which re-enables non-critical services.`,
+	Example: `  system clearbusy`,
+	Args:    cobra.NoArgs,
+	RunE:    withClient(clearBusyCmdF),
+}
+
+func init() {
+	SystemSetBusyCmd.Flags().UintP("seconds", "s", 3600, "Number of seconds until server is automatically marked as not busy.")
+	_ = SystemSetBusyCmd.MarkFlagRequired("seconds")
+	SystemCmd.AddCommand(
+		SystemGetBusyCmd,
+		SystemSetBusyCmd,
+		SystemClearBusyCmd,
+	)
+	RootCmd.AddCommand(SystemCmd)
+}
+
+func getBusyCmdF(c client.Client, cmd *cobra.Command, _ []string) error {
+	printer.SetSingle(true)
+
+	sbs, response := c.GetServerBusy()
+	if response.Error != nil {
+		return fmt.Errorf("unable to get busy state: %w", response.Error)
+	}
+	printer.PrintT("busy:{{.Busy}} expires:{{.Expires}}", sbs)
+	return nil
+}
+
+func setBusyCmdF(c client.Client, cmd *cobra.Command, args []string) error {
```

Comment:
We should add `printer.SetSingle(true)` to this command too

Reply:
Fixed via https://github.com/mattermost/mmctl/pull/130/commits/cb820ee21914e2fa5792285f7538644a9f79f523